tests/Unit/Core/Business/Checkout/TermsAndConditionsTest.php

```diff
@@ -16,15 +16,15 @@ public function setup()
     public function test_SetText_InsertsLinks()
     {
         $this->assertEquals(
-            'hello <a href="http://www.world.com">world</a>',
+            'hello <a href="http://www.world.com" id="cta--0">world</a>',
```

Comment:
@mickaelandrieu 
you are right... I adapted the tests again: 
https://github.com/PrestaShop/PrestaShop/commit/7abffa419b4d3b8f7381308e3cbe7bb03b01fec7